lib/src/realm_class.dart

```diff
@@ -486,10 +487,20 @@ class Realm implements Finalizable {
     return realmCore.realmEquals(this, other);
   }
 
-  /// The logger to use for logging
-  static Logger logger = Logger.detached('Realm')
-    ..level = RealmLogLevel.info
-    ..onRecord.listen((event) => print(event));
+  /// The logger to use for logging.
+  /// The default logger is [RealmLogger] and the default log level is [RealmLogLevel.info].
```

Comment:
I don't think we should mention `RealmLogger`.

Reply:
@nielsenko thank you about your comments. All your suggestions are handled in https://github.com/realm/realm-dart/pull/1226/commits/5da5b375f813574d63eaaae13709cae28a847e7b commit.